.golangci.yml

```diff
@@ -0,0 +1,28 @@
+run:
+  deadline: 10m
+
+output:
```

Comment:
I suggest removing this, [rationale here](https://github.com/decred/dcrvotingweb/pull/290#discussion_r1229206494)